tests/pfcwd/test_pfcwd_timer_accuracy.py

```diff
@@ -350,21 +350,18 @@ def retrieve_timestamp(self, pattern):
             syslog_msg = self.dut.shell(cmd)['stdout']
 
             # Regular expressions for the two timestamp formats
-            regex1 = re.compile(r'^[A-Za-z]{3} \d{2} \d{2}:\d{2}:\d{2}\.\d{6}')
-            regex2 = re.compile(r'^\d{4} [A-Za-z]{3}\s{1,2}\d{1,2} \d{2}:\d{2}:\d{2}\.\d{6}')
-
-            if regex1.match(syslog_msg):
-                timestamp = syslog_msg.replace('  ', ' ').split(' ')[2]
-            elif regex2.match(syslog_msg):
-                timestamp = syslog_msg.replace('  ', ' ').split(' ')[3]
+            regex = re.compile(r'\b[A-Za-z]{3}\s{1,2}\d{1,2} \d{2}:\d{2}:\d{2}\.\d{6}\b')
```

Comment:
For spaces between month and day part: 
Overly rigid whitespace handling. The pattern \s{1,2} only allows 1 or 2 spaces, but some log formats may use tabs or more spaces, causing potential mismatches.
suggest to use "\s+"

Reply:
Thanks for your suggestion.
The timestamp format should be fixed, the current format should be "%Y %b %d %H:%M:%S.%f". 
I ensure the %d would be 2 \d before, but actually, it would output " 1" instead of "01", so modify the regex to \s{1,2}\d{1,2} by PR https://github.com/sonic-net/sonic-mgmt/pull/16757.
I don't think it's necessary to use "\s+", thanks.